docs/editors/vim.md

```diff
@@ -99,6 +97,9 @@ set shortmess+=c
 " Always show signcolumns
 set signcolumn=yes
 
+" Help Vim recognize *.sbt and *.sc as Scala files
+au BufRead,BufNewFile *.sbt,*.sc set filetype=scala
```

Comment:
Everything above is just formatting changes (just text width), and I moved this section from above to here and added the `*.sc`.

Reply:
checking docs: https://scalameta.org/metals/docs/editors/vim.html
should you not update this?
`au BufRead,BufNewFile *.sbt set filetype=scala`

Reply:
ah nvm you moved it, sry :facepalm: